### PR TITLE
Using WDI For Writing Claims + General Code Cleanup

### PR DESCRIPTION
**Pull request by kennethsn**
This PR does the following:
- Using WikidataIntegrator instead of PyWikiBot for writing claims
- Removing PyWikiBot as a dependency
- Adding Support For Dev Wikibase instance (currently pretty minimal)
- Continue Fixing PyLint Warnings
- Ensuring all tests pass

TODO's for Future PR's:
- Integrating schemas with Dev Wikibase Properties
- Update SPARQL Queries to take this into account Dev Wikibase
- Update Qlabel to support Wikibase

### CODE REVIEW

File: wikidp/config.py [hunk 1]
```diff
@@ -47,33 +54,31 @@ class DevConfig(BaseConfig):
...
 CONFIGS = {
     "dev": 'wikidp.config.DevConfig',
     "default": 'wikidp.config.BaseConfig'
 }
 
+
 def configure_app(app):
     """Grab the environment variable for app config or defaults to dev."""
-    config_name = os.getenv('WIKIDP_CONFIG', 'dev')
+    config_name = os.getenv('WIKIDP_CONFIG', 'default')
```
Comment (carlwilson) [inline wikidp/config.py:69]: The clue was in the name :)

File: wikidp/controllers/search.py [hunk 2]
```diff
@@ -45,26 +49,28 @@ def get_search_result_context(search_string):
         for res in FileFormatExtSearchResult.search(search_string)
     ]
     # Check if searching with PUID
-    # pylint: disable=W1401
-    if re.search("[x-]?fmt/\d+", search_string) is not None:
+    if re.search(r"[x-]?fmt/\d+", search_string) is not None:
```
Comment (carlwilson) [inline wikidp/controllers/search.py:52]: regex here could probably be a named constant for clarity

File: wikidp/utils/__init__.py [hunk 3]
```diff
@@ -220,38 +249,43 @@ def get_lang(_dict, default=None):
...
     item = wd_int_utils.get_item_json(qid)
     if not item:
         return False
-    label = parse_wd_response_by_key(item, 'labels', default="Item {}".format(qid))
+    label = parse_wd_response_by_key(item, 'labels', default=f"Item {qid}")
     aliases = parse_wd_response_by_key(item, 'aliases', default=[])
     description = parse_wd_response_by_key(item, 'descriptions', default='')
-    output_dict = {'qid': qid, 'label': label, 'aliases': aliases, 'description': description}
+    output_dict = {
+        'aliases': aliases,
```
Comment (carlwilson) [inline wikidp/utils/__init__.py:276]: The key names here might be good candidates for constants also

Comment (kennethsn) [reply wikidp/utils/__init__.py:276]: Agreed!
I'm planning to refactor quite a bit of this file to make the code a bit easier to follow as well (more object-oriented). I'll be sure to keep this comment in mind when I do!

File: wikidp/utils/__init__.py [hunk 4]
```diff
@@ -192,16 +206,31 @@ def get_wikimedia_image_url_from_title(title):
         return "https://commons.wikimedia.org/wiki/File:"+title
```
Comment (carlwilson) [inline wikidp/utils/__init__.py:206]: again here with the the constants.

File: wikidp/controllers/search.py (hunk 2 above)
Comment (kennethsn) [reply wikidp/controllers/search.py]: Good catch, just updated!

File: wikidp/utils/__init__.py (hunk 4 above)
Comment (kennethsn) [reply wikidp/utils/__init__.py]: Just updated!

**Pull request merged**